extensions/wikia/Forum/ForumController.class.php

```diff
@@ -289,17 +323,22 @@ protected function getSortingSelected() {
 		$selected = $this->wg->request->getVal( 'sort' );
 
 		if ( empty( $selected ) ) {
-			$selected = $this->app->wg->User->getGlobalPreference( 'forum_sort_' . $this->sortingType );
+			$selected =
+				$this->app->wg->User->getGlobalPreference( 'forum_sort_' . $this->sortingType );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "forum_sort_" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)